src/algebra/ordered_monoid.lean

```diff
@@ -75,6 +75,70 @@ ordered_comm_monoid.mul_le_mul_left a b h c
 lemma mul_le_mul_right' (h : a ≤ b) (c) : a * c ≤ b * c :=
 by { convert mul_le_mul_left' h c using 1; rw mul_comm }
 
+@[to_additive]
+lemma mul_lt_of_mul_lt_left (h : a * b < c) (hle : d ≤ b) : a * d < c :=
+lt_of_le_of_lt (mul_le_mul_left' hle a) h
```

Comment:
```suggestion
(mul_le_mul_left' hle a).trans_lt h
```
`lt_of_le_of_lt` has a shorthand as `has_le.le.trans_lt`

Reply:
Thanks! I always forget about this one.